packages/osd-optimizer/src/common/theme_tags.test.ts

```diff
@@ -35,6 +35,8 @@ it('returns default tags when passed undefined', () => {
     Array [
       "v7dark",
       "v7light",
+      "v8dark",
+      "v8light",
```

Comment:
The `v8` prefix is meaningless. Because of the selection logic, I could name it absolutely anything.
```js
// Next (preview) label is mapped to v8 here
const themeTag = `${themeVersion === 'v7' ? 'v7' : 'v8'}${darkMode ? 'dark' : 'light'}`;
```

`themeVersion` is the value of `theme:version` config which is defined as `['v7', 'Next (preview)']` and the code above, doesn't use it as is but rather translates it to a short prefix perhaps because the schema is incapable of holding labels as well as values. This logic also limits the system to just 2 themes at a time!

A future enhancement could simply replace non-alphanumeric characters in the theme name with a hyphen and make it lower-case to be used as a prefix; this will allow OSD to entertain more than two themes at a time.